### DISCUSSION TITLE
Tell me your favorite artist!!!!!

### DISCUSSION THREAD

**Discussion by ShuheiKosaka** [NAIST-SE/AttractingContributors#1087] [2024 Attraction Contributions]
Click here!!
https://github.com/ShuheiKosaka/Music

**Comment by gill876** [comment]
Ariana Grande!

Please add your hobby to my README:
https://github.com/gill876/AttractingContributors-Hiking?tab=readme-ov-file

**Comment by oyewaleoyebode** [comment]
David Adeleke

**Comment by ryosuke-sota** [comment]
I made a PR on your repo. </br>Could you contribute to mine? </br>Here : [my project link (fav drinks)](https://github.com/ryosuke-sota/AttractingContributors.git)  </br>Thank you.

**Comment by 3207-Rhims** [comment]
I made a PR on you. Can you make one for me. https://github.com/NAIST-SE/AttractingContributors/compare/main...3207-Rhims:AttractingContributors-Movie:main?expand=1

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！
